statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1287,17 +1461,20 @@ def reorder(self, order):
 #-------------------------------------------------------------------------------
 # VAR Diagnostics: Granger-causality, whiteness of residuals, normality, etc.
 
-    def test_causality(self, equation, variables, kind='f', signif=0.05,
-                       verbose=True):
-        """Compute test statistic for null hypothesis of Granger-noncausality,
-        general function to test joint Granger-causality of multiple variables
+    def test_causality(self, causing, kind='f', signif=0.05, verbose=True):
```

Comment:
Thanks, Josef, for pointing this out!
How does the deprecation process work? Because I really feel like the API should be changed in future.
Here is a [Jupyter-notebook](https://gist.github.com/yogabonito/eedd44be9b8a76de7773a1e43f961d45) showing that specifying `causing` _and_ `caused` may give unexpected results - more precisely: if the union of these arguments is not the whole system of variables (this is the case in section 1 of the notebook and the results differ from those in section 2 although the causality test is done using the same variables).
[Note: the notebook was produced using the latest version of var_model.py before I changed the file for the first time - so the method's signature includes both `causing` and `caused` (called `variables` and `equation` in the old code)]